packages/openssl/cryto-ec-asm-ecp_nistz256-armv4.pl.patch

```diff
@@ -0,0 +1,11 @@
+--- ./crypto/ec/asm/ecp_nistz256-armv4.pl	2021-09-07 17:16:32.000000000 +0530
++++ ./crypto/ec/asm/ecp_nistz256-armv4.pl.mod	2021-10-04 16:51:34.391932913 +0530
+@@ -81,7 +81,7 @@
+ die "insane number of elements" if ($#arr != 64*16*37-1);
+ 
+ $code.=<<___;
+-.rodata
++.section .rodata
```

Comment:
We don't need this additional patch. `ASFLAGS+=" -fno-integrated-as"` fixed the build.

Reply:
I force pushed just the moment you wrote this comment 😆. Btw thanks because I didn't do this before your previous comment.